src/revolution/dvd/dvdFatal.c

```diff
@@ -0,0 +1,99 @@
+#include "revolution/dvd.h"
+#include "revolution/os.h"
+#include "revolution/sc.h"
+
+static funcptr_t FatalFunc = NULL;
+
+//! TODO: fix encoding issues
```

Comment:
what are the encoding issues? should it be utf-8 instead of shift-jis or something?

Reply:
the section is matched but I needed to hardcode some weird chars for that